debug.py

```diff
@@ -0,0 +1,24 @@
+from traceback import print_exc
+
+
+
+try: __debugload__
+except:   
+    __debugload__ = True
+    
+    debug = False
+    
+    def printdebug(str, msgdebug):

```

Comment:
move to RseData

Reply:
and remove `global`

Reply:
Just an idea: Add "EDSM-RSE" in the debug output function. That way there is only 1 place where it has to be changed and also we don't forget to include it in the message